### ISSUE TITLE
Count fetching method

### ISSUE THREAD

**Issue by frbuceta** [nuxt/content#378]
### Is your feature request related to a problem? Please describe.

I just started with `@nuxt/content` and I am doing a pagination but I have seen that there is no count method.

### Describe the solution you'd like

My idea would be the following:
- Create a count method that is a replacement for fetch and will return a Promise <number>
- Also, only the where and search methods will be supported (See example below)

### Describe alternatives you've considered

I haven't considered any more ideas

### Additional context

Example:
```js
await this.$content('articles')
          .where()
          .search()
          .count()

// Returns: 30
```

**Comment by mathe42** [comment]
You can use `(await $content('blog').only([]).fetch()).length` but your feature request helps to minimize Loading data

**Comment by zarub2k** [comment]
I need this functionality for my pagination implementation. This would be a great help if I just need the total number of articles without loading any data. The actual data will be loaded based on the number of items I need per page based on the current page I am in.

**Comment by Geminii** [comment]
I really like this idea to permit multiples features like pagination :) 
I will purpose a PR in this direction 😀

**Comment by farnabaz** [comment]
The `count` helper is not available in v2.8. https://content.nuxtjs.org/api/composables/query-content#count
